mkdocs.yaml

```diff
@@ -37,14 +37,14 @@ extra_css:
 
 nav:
 - Getting started:
-  - Overview: overview.md
+  - Overview: index.md
```

Comment:
FYI, I found the alternative file name necessary to control what's displayed in the left-navigation menu. If you figured out another way, that's better.

Reply:
i agree that this is suboptimal behavior. however, if we don't have an index, the main page is broken and clicking on the link to the main site leads to 404. i think i know how to fix this, this will come later